test/Factory/ContainerAbstractServiceFactoryTest.php

```diff
@@ -41,14 +41,16 @@ public function testInstantiate(): void
             ->with(ManagerInterface::class)
             ->willReturn(true);
 
+        $managerMock = $this->createMock(ManagerInterface::class);
+
         $this->container->expects($this->once())
             ->method('get')
             ->willReturnMap([
-                [ManagerInterface::class => ManagerInterface::class],
+                [ManagerInterface::class, $managerMock],
```

Comment:
```suggestion
                [ManagerInterface::class, $this->createMock(ManagerInterface::class)],
```
Your solution was correct, but as a general rule:
> Unless you want to reuse something, you don't need to put it in its own variable.